joss_paper_0.18/paper.bib

```diff
@@ -0,0 +1,38 @@
+@SciPy{,
+  author =    {Eric Jones and Travis Oliphant and Pearu Peterson and others},
```

Comment:
This is taken from http://scipy.org/citing.html. Should we keep it like that, or update according to the author ranking of Zenodo for example? This is not an actual fixed citation after all.